static/src/stylesheets/layout/new-header/_subnav.scss

```diff
@@ -7,7 +7,6 @@
 
     @include mq(mobile) {
         max-height: $gs-baseline * 3.5;
-        max-height: 13vw;
```

Comment:
This didn't seem necessary to me. But maybe @gustavpursche @zeftilldeath can let me know if it is